### ISSUE TITLE
PDF prospectus design

### ISSUE THREAD

**Issue by jonitrythall** [djangocon/2020.djangocon.us#7]

**Comment by jefftriplett** [comment]
@jonitrythall you asked for copy for a letter in the prospectus but I think I have blank page anxiety. Any examples or tips so I know where to begin?

**Comment by jonitrythall** [comment]
@jefftriplett Of course! [Here's an example of Ela Conf's] on page 2 (https://www.dropbox.com/s/7b4cs19eyy6osbw/ela-prospectus-2017%20copy.pdf?dl=0). I think the main thing is speaking directly to them and sharing any info they would be especially interested in from a business/marketing perspective

**Comment by jefftriplett** [comment]
Thank you!
